### PR TITLE
[2.0] Drop commands to generate repository and entity stubs

### PR DESCRIPTION
**Pull request by alcaeus**
Deprecation in 1.3 will come once we have the appropriate branch.

### CODE REVIEW

File: lib/Doctrine/ODM/MongoDB/Tools/Console/Command/GenerateDocumentsCommand.php [hunk 1]
```diff
@@ -1,146 +0,0 @@
...
-            );
-        } elseif ( ! is_writable($destPath)) {
-            throw new \InvalidArgumentException(
-                sprintf("Documents destination directory '<info>%s</info>' does not have write permissions.", $destPath)
-            );
-        }
-
-        if (count($metadatas)) {
-            // Create DocumentGenerator
-            $documentGenerator = new DocumentGenerator();
```
Comment (malarzm) [inline lib/Doctrine/ODM/MongoDB/Tools/Console/Command/GenerateDocumentsCommand.php]: `DocumentGenerator` and related tests should be removed too, no?

Comment (alcaeus) [reply lib/Doctrine/ODM/MongoDB/Tools/Console/Command/GenerateDocumentsCommand.php]: Good point

**Comment by malarzm** [comment]
Almost -2k - that's a gain! FTR I can't merge due to conflict :(

**Comment by alcaeus** [comment]
My bad - fixed now.

**Pull request merged**